.github/workflows/training-model-mgmt-unittests.yaml

```diff
@@ -0,0 +1,53 @@
+name: training-model-mgmt-unittests
+
+on:
+  # push:
+  #   branches:
+  #     - release
+  pull_request:
+    branches:
+      - main
+      - ayushmishra/update_mlflow_convertor_params
+    paths:
+      - assets/training/model_management/*
```

Comment:
Include common/deploy or registration as well?

Reply:
updated

Reply:
For register and model-local-validation